doc/openapi-apicurio.md

```diff
@@ -0,0 +1,79 @@
+## Usin APICurio to edit an OpenAPI spec and visualise Kuadrant extensions
+
+This guide assumes you have APICurio already running.
+See https://www.apicur.io/ for info on how to install APICurio Studio.
+
+### Editing the spec
```

Comment:
Before diving into editing, might be worth a pre-amble on OAS extensions, linking out to https://swagger.io/docs/specification/openapi-extensions/